src/components/Tooltip/Tooltip.tsx

```diff
@@ -17,15 +18,24 @@ function customArrow(placement: Placement, arrowProps) {
   );
 }
 
-export interface TooltipProps extends CoreTooltipProps {}
+export interface TooltipProps extends CoreTooltipProps {
+  skin?: TooltipSkin;
```

Comment:
If you would add a comment here it will update the storybook API tab

Reply:
Added.